crates/code_generation/src/accessor.rs

```diff
@@ -287,14 +287,7 @@ mod tests {
         let cycler = Cycler {
             name: "TestCycler".to_string(),
             kind: CyclerKind::RealTime,
-            instances: vec![
-                Instance {
-                    name: "InstanceA".to_string(),
-                },
-                Instance {
-                    name: "InstanceB".to_string(),
-                },
-            ],
+            instances: vec!["InstanceA".to_string(), "InstanceB".to_string()],
```

Comment:
Maybe we already talked about this, is it possible to take `&'static str` instead of `String` here?

Reply:
We talked about this for the manifest, however, this isn't the `CyclerManifest` type but the `Cycler` type.
It shouldn't normally be constructed manually anyways.

Besides, since the instance names are compounds of cycler name and user provided instance names, the new string has to be stored somewhere. If we were to to use string slices here, something else would have to own that string.